### PR TITLE
Kevinsikes dearborn

### PR DESCRIPTION
**Pull request by kevinsikes**
Some of my favorite nearby restaurants.  Merge so others can enjoy!

### CODE REVIEW

File: Burgers.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Five Guys
```
Comment (githubteacher) [inline Burgers.txt:1]: What are your thoughts on Brome Burgers?

File: Mediterranean.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Golden Chicken Grill, 22815 Michigan Ave, Dearborn, MI 48124
+Sandwich, soup or salad, bread & garlic sauce, and drink for $6.99
```
Comment (githubteacher) [inline Mediterranean.txt:2]: What about La Pita?

File: Burgers.txt (hunk 1 above)
Comment (devops23345) [inline Burgers.txt:1]: What about Millers in Dearborn?

**Review by githubteacher — COMMENTED** [review]
Just some ideas on places to eat, do you want to add them to your list @kevinsikes

**Comment by kevinsikes** [comment]
I like your additions and will make the changes

File: Mediterranean.txt (hunk 2 above)
Comment (kevinsikes) [reply Mediterranean.txt:2]: Great suggestion @githubteacher !

**Pull request merged**